src/app/shared/dockstore.model.ts

```diff
@@ -33,6 +33,7 @@ export class Dockstore {
   static DNANEXUS_IMPORT_URL = 'https://platform.dnanexus.com/panx/tools/import-workflow';
   static TERRA_IMPORT_URL = 'https://app.terra.bio/#import-tool/dockstore';
   static CGC_IMPORT_URL = 'https://cgc.sbgenomics.com/integration/trs/import';
+  static ANVIL_IMPORT_URL = 'https://anvil.terra.bio/#import-tool/dockstore';
```

Comment:
FYI to other reviewers: checked with Broad, and they do not have a staging instance of Anvil, so there will probably not be corresponding web service and compose_setup PRs to configure this.